### PR TITLE
Update shorah

### PR DESCRIPTION
**Pull request by DrYak**
ShoRAH 2 - introducing version v1.99.0
ShoRAH 1 - fixing builds of 1.1.3 and deprecating 1.1.0 and 1.0.0
de-blacklisting

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title
* New recipes not directly relevant to the biological sciences need to be submitted to [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please add the `please review & merge` label.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>BiocondaBot commands</summary>
      
Everyone has access to the following BiocondaBot commands, which can be given in a comment:

  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
  * `@BiocondaBot please add label` will add the `please review & merge` label.
  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.

For members of the Bioconda project, the following command is also available:

 * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
</details>

### CODE REVIEW

**Comment by DrYak** [comment]
@BiocondaBot please add label

**Comment by DrYak** [comment]
@BiocondaBot please fetch artifacts

File: recipes/shorah/1.99.0/meta.yaml [hunk 1]
```diff
@@ -32,8 +33,6 @@ requirements:
     - gsl
     - zlib
     - blas {{ blas }}
-    - setuptools
-    - pip
```
Comment (DrYak) [inline recipes/shorah/1.99.0/meta.yaml]: Saddly, this [breaks the build on mac OS X](https://circleci.com/gh/bioconda/bioconda-recipes/101896) for some reason (badly written setup.py ?).

I'll try to get our devs to fix that, and will get back to you.

**Comment by DrYak** [comment]
Latest edit break building on mac OS X.
I need to remove the label and address the problem.
How could I do that ?

@BiocondaBot please remove label

**Comment by DrYak** [comment]
@BiocondaBot please fetch artifacts

File: recipes/shorah/1.99.0/meta.yaml [hunk 2]
```diff
@@ -32,6 +32,8 @@ requirements:
     - numpy
     - gsl
     - zlib
+    - setuptools
```
Comment (DrYak) [inline recipes/shorah/1.99.0/meta.yaml:35]: setuptools _should_ suffice for now.

Comment (dpryan79) [reply recipes/shorah/1.99.0/meta.yaml:35]: OK, go ahead and remove pip and I'll approve the PR so it can get merged.

Comment (DrYak) [reply recipes/shorah/1.99.0/meta.yaml:35]: Thanks!

**Comment by DrYak** [comment]
@dpryan79 here it is

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**